xmppserver/src/main/java/org/jivesoftware/util/S2STestService.java

```diff
@@ -154,15 +171,14 @@ private void logSessionStatus() {
         final DomainPair pair = new DomainPair(XMPPServer.getInstance().getServerInfo().getXMPPDomain(), domain);
         OutgoingServerSession session = XMPPServer.getInstance().getSessionManager().getOutgoingServerSession(pair);
         if (session != null) {
-            int connectionStatus = session.getStatus();
-            switch(connectionStatus) {
-            case Session.STATUS_CONNECTED:
+            switch(session.getStatus()) {
```

Comment:
Is this switch statement necessary? Would not `log.info("Session is {}", session.getStatus())`; suffice?

Reply:
Good catch. I'll add that.